chpl/chpl-service/src/main/java/gov/healthit/chpl/web/controller/CertifiedProductController.java

```diff
@@ -143,6 +151,43 @@
 
         return certifiedProduct;
     }
+    
+    @ApiOperation(value = "Download all SED details that are certified to 170.315(g)(3).",
+            notes = "Download a specific file that is generated overnight.")
+    @RequestMapping(value = "/sed_details", method = RequestMethod.GET)
+    public void streamSEDDetailsDocumentContents(HttpServletResponse response) throws EntityRetrievalException, IOException {
+    	Path path = Paths.get(env.getProperty("downloadSEDFolderPath"));
```

Comment:
I thought you took this property out? Because you'd have the filename in both the file creation place and in this file download place, the file name actually *should* be a property. Just the name of the SED download file, not the full path. The full path can be constructed by putting together the downloadFolderPath property and the new sedFilename property (or whatever) that you will add. Make sure to change the file generation script to read it as well.